src/accelerate/optimizer.py

```diff
@@ -102,14 +102,14 @@ def zero_grad(self, set_to_none=None):
                 set_to_none = False
             self.optimizer.zero_grad(set_to_none=set_to_none)
 
-    def step(self):
+    def step(self, closure=None):
         if self.state.distributed_type == DistributedType.TPU:
-            xm.optimizer_step(self.optimizer)
+            xm.optimizer_step(self.optimizer, optimizer_args={"closure": closure})
```

Comment:
Actually thinking a little bit more about it, let's not pass anything if there is no closure:
```suggestion
            optimizer_args = {"closure": closure} if closure is not None else {}
            xm.optimizer_step(self.optimizer, optimizer_args=optimizer_args)
```

Reply:
I concur, this is a little more explicit, but it shouldn't matter (famous last words...) because the base `torch.optim.Optimizer` has a `closure` argument. For the other options, we pass that along anyway.

Reply:
If you don't mind committing the suggestion then, we're all good to merge :-)